wire-ios-utilities/Source/Keychain/Keychain.swift

```diff
@@ -61,6 +65,8 @@ public enum Keychain {
     }
 
     public static func deleteItem<T: KeychainItem>(_ item: T) throws {
+        WireLogger.keychain.info("[Keychain] deleting item (\(item))")
```

Comment:
`"[Keychain] "` is not required, right? The `tag` argument contains `"keychain"`.

```
public extension WireLogger {
    static let keychain = WireLogger(tag: "keychain")
}
```